log/standardLogger.go

```diff
@@ -113,12 +111,17 @@ func (sl *standardLogger) PutFields(fields frozen.Map) Logger {
 }
 
 func (sl *standardLogger) SetFormatter(formatter Config) error {
+	pkgFormatter, isPkgFormatter := formatter.(Formatter)
+	if isPkgFormatter {
+		sl.internal.SetFormatter(&pkgToLogrusFormatter{pkgFormatter})
+		return nil
+	}
 	logrusFormatter, isLogrusFormatter := formatter.(logrus.Formatter)
-	if !isLogrusFormatter {
-		return errors.New("formatter is not logrus formatter type")
+	if isLogrusFormatter {
+		sl.internal.SetFormatter(logrusFormatter)
+		return nil
 	}
-	sl.internal.SetFormatter(logrusFormatter)
-	return nil
+	return errors.New("formatter is not logrus formatter type")
```

Comment:
maybe change the error message to `formatter is not Formatter or logrus.Formatter`

Reply:
Good.